opencog/nlp/learn/run/observe-launch.scm

```diff
@@ -0,0 +1,50 @@
+;
+; observe-launch.scm
+;
+; Run the cogserver, needed for the language-learning disjunct
+; counting pipeline. Starts the cogserver, opens the database,
+; loads the database (which can take an hour or more!)
+;
+(use-modules (opencog) (opencog persist) (opencog persist-sql))
+(use-modules (opencog cogserver))
+(use-modules (opencog nlp) (opencog nlp learn))
+(use-modules (system repl common))
+(use-modules (system repl server))
+
+(repl-default-option-set! 'prompt "scheme@(en-pairs)> ")
```

Comment:
well, the point of having the prompt in a config file is to have differrent prompts for different languages -- they are not all going to be 'en', and it can get very confusing if you are processing multiple languages at the same time. This really belongs back in the config file, like it used to be.

Also -- different stages -- parsing, computing the MI, performing the MST, performing the disjunct extraction, and performing the merges -- these all should get different prompts so tht one can tell which is which from the prompt. (so, for one language - english - there will be five different prompts.